Formula/libpeas.rb

```diff
@@ -4,6 +4,7 @@ class Libpeas < Formula
   url "https://download.gnome.org/sources/libpeas/1.34/libpeas-1.34.0.tar.xz"
   sha256 "4305f715dab4b5ad3e8007daec316625e7065a94e63e25ef55eb1efb964a7bf0"
   license "LGPL-2.1-or-later"
+  revision 1
```

Comment:
I'm actually triggering a rebottle of `libpeas`. It was incompletely migrated to Python 3.11 probably due to Python 3.10 pkgconfig files (e.g. looks for `python3-embed` which is only available for aliased Python).

Reply:
that makes better sense.